File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1039.rb
while linha = gets
    ls = linha.split(' ')
        r1 = (ls[0]).to_i
        x1 = (ls[1]).to_i
        y1 = (ls[2]).to_i
        r2 = (ls[3]).to_i
        x2 = (ls[4]).to_i
        y2 = (ls[5]).to_i
        
        d = (x1-x2)*(x1-x2) + (y1-y2)*(y1-y2)
        
    if(d <= r1*r1 and Math.sqrt(d)+r2 <= r1) then puts("RICO")
    else puts("MORTO") end
end

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1039.lua
function split(inputstr, sep)
    if sep == nil then
            sep = "%s"
    end
    local t={}
    for str in string.gmatch(inputstr, "([^"..sep.."]+)") do
            table.insert(t, str)
    end
    return t
end

while true do
    linha = io.read()
    if linha == nil then break end
    ls = split(linha)

    r1 = tonumber(ls[1])
    x1 = tonumber(ls[2])
    y1 = tonumber(ls[3])
    r2 = tonumber(ls[4])
    x2 = tonumber(ls[5])
    y2 = tonumber(ls[6])
    
    d = (x1-x2)*(x1-x2) + (y1-y2)*(y1-y2)
        
    if(d <= r1*r1 and math.sqrt(d)+r2 <= r1) then print("RICO")
    else print("MORTO") end

end